fw/tls.c

```diff
@@ -333,20 +342,23 @@ tfw_tls_encrypt(struct sock *sk, struct sk_buff *skb, unsigned int mss_now,
 	t_sz = skb_tail->truesize;
```

Comment:
It seems `tfw_tls_encrypt()` also make a side effect in `tfw_tcp_propagate_dseq()` at the above.

In general: do we need the new complexity, isn't it easier to not to call `sk_write_xmit()` in the kernel and just reset the TCP connection? If we failed with the current skb encryption, what we're expecting to get with the next call? In my understanding, we'll just either get corrupted application data or terminate the connection anyway.